recap-project-4/src/App.js

```diff
@@ -1,16 +1,31 @@
-
-import './App.css';
+// import './App.css';
+import Form from './components/Form/index';
+// import {useState} from "react" Aufgabe 1
+import {uid} from "uid"
+import List from './components/List/index';
+import useLocalStorageState from "use-local-storage-state";
 
 function App() {
+  // Why setting state in App.js? Possible answer: Parent element of Form.js. Here we want to change the state while
+  // displaying the new activity
+  //  const [activities, setActivities] = useState([]); Aufgabe 1//
+   const [activities, setActivities] = useLocalStorageState("activities", {defaultValue: []});
+
+   //here we set up the new activity
+   function handleAddActivity(newActivity) {
+    //understanding of the following line: the function setActivities is setting the newActivity, passing a unique id to it while creating 
+    // an object and adding with the spread operator the newActivity to the other activities into an array of objects
+      setActivities([{newActivity, id: uid()}, ...activities])
```

Comment:
you probably meant to spread the newActivity here

```js
setActivities([{...newActivity, id: uid()}, ...activities])
```

Otherwise it will create a key `newActivity` that has an object with the form data as its value